plugin/trino-iceberg/src/test/java/io/trino/plugin/iceberg/BaseIcebergConnectorTest.java

```diff
@@ -1143,6 +1143,69 @@ public void testEmptySortedByList()
         dropTable(tableName);
     }
 
+    protected abstract boolean isFileSorted(String path, String sortColumnName);
+
+    @Test
+    public void testSortedNationTable()
```

Comment:
could you pls add an extra test for sort order evolution:

create table t (a integer, b integer ) with (sort_order=array['a']);
insert into t values (1,100), (3, 3), (2,20)
alter table t set properties sorted_by = array['b'];
insert into t values ....

verify that the first file is sorted by "a" and that the second file is sorted by "b".